File: June-Leetcoding-Challenge/9. Is Subsequence/subsequence.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        if len(s) == 0 :
            return True
        i = 0
        exist = 0
        for letter in t:
            if i < len(s):
                if s[i] == letter:
                    exist += 1
                    i = i + 1
                else:
                    continue
            else:
                break
        if len(s) == exist:
            return True
        return False

File: June-Leetcoding-Challenge/9. Is Subsequence/subsequence.java
class Solution {
    public boolean isSubsequence(String s, String t) 
    {
        if(s.length() == 0)
            return true;
        
        int i = 0, exist = 0;
        for(char letter: t.toCharArray()){
            if(i < s.length())
            {
                if(s.charAt(i) == letter)
                {
                    exist += 1;
                    i += 1;
                }
                else
                    continue;
            }
            else
                break;
        }
        if(exist == s.length())
            return true;
        return false;
    }
}